source/github-helpers/get-comment-author.ts

```diff
@@ -0,0 +1,36 @@
+/**
+Given any element in a comment, returns the comment’s author
+
+This works on:
+- First comment
+- Following comments
+- Main review comment
+- Following review comments
+- Gist comments
+- Bots
+
+Note: Bots are used as `name[bot]`, `app/name`, or `apps/name` depending on the context:
+- https://github.com/webpack/webpack/commits?author=dependabot%5Bbot%5D
+- https://github.com/webpack/webpack/pulls/app%2Fdependabot
+- https://github.com/apps/dependabot
+
+@returns user-name
+@returns dependabot[bot]
+
+*/
+export default function getCommentAuthor(anyElementInsideComment: Element): string {
```

Comment:
Can we reserve this helper for `show-names`?

Reply:
That was the idea, just not as part of this PR

Reply:
2 great minds think alike 😜

Reply:
Added to #5222

Reply:
`show-names` has to query the exact author link so this function is not useful. I don't see any other situations where it can be used (grepped `author`)